### PR TITLE
✨ Change date and time picker to material design

### PR DESCRIPTION
**Pull request by panwar2001**
### DATE & TIME PICKER

solution for issue #601

### CODE REVIEW

**Comment by panwar2001** [comment]
what else can be done?
@LeoColman

**Comment by LeoColman** [comment]
Thanks for the contribution @panwar2001 !! It's awesome to have people interested in this project :)

One thing I'd like to address before merging the PR and taking it off your hands is removing the 12H version of the TimeDialog and keep only the 24H version for now. We have issue #618 to take care of letting the user choose which format they preffer.

**Comment by panwar2001** [comment]
### 24 hour mode time picker

File: app/src/main/kotlin/br/com/colman/petals/components/TimePicker.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.TimePicker
+import androidx.compose.material3.rememberTimePickerState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import br.com.colman.petals.R.string.cancel
+import br.com.colman.petals.R.string.ok
+import br.com.colman.petals.R.string.select_time
+import com.vanpra.composematerialdialogs.MaterialDialogState
```
Comment (LeoColman) [inline app/src/main/kotlin/br/com/colman/petals/components/TimePicker.kt:15]: We are still using this guy around.
Is this a missimport? We should be able to ditch vanpra dialogs altogether

**Review by LeoColman — CHANGES_REQUESTED** [review]

**Comment by panwar2001** [comment]
Removed the vanpra library usage 
but , there is a warning message in other files as material and material3 library is used together.

image

File: app/src/main/kotlin/br/com/colman/petals/components/TimePicker.kt (hunk 1 above)
Comment (panwar2001) [reply app/src/main/kotlin/br/com/colman/petals/components/TimePicker.kt]: okay

**Review by LeoColman — APPROVED** [review]

**Comment by LeoColman** [comment]
I think we are OK on having those warnings until we migrate all components to Material3.

Thanks for the contribution @panwar2001 !!

**Pull request merged**